docker/rootfs/usr/local/go2rtc/create_config.py

```diff
@@ -64,8 +64,21 @@
 # as source for frigate and the integration supports HLS playback
 if go2rtc_config.get("rtsp") is None:
     go2rtc_config["rtsp"] = {"default_query": "mp4"}
-elif go2rtc_config["rtsp"].get("default_query") is None:
-    go2rtc_config["rtsp"]["default_query"] = "mp4"
+else:
+    if go2rtc_config["rtsp"].get("default_query") is None:
+        go2rtc_config["rtsp"]["default_query"] = "mp4"
+
+    # perform environment variable substitution for the go2rtc rtsp creds
+
```

Comment:
```suggestion

```

Reply:
One more quick change then I think it should be good

Reply:
Done